### PR TITLE
Restore ADO pipeline triggers

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: src/uk/gov/defra/ffc/ADO.groovy [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
   static void triggerPipeline(def ctx, String namespace, String chartName, String chartVersion, Boolean hasDatabase) {
-    if (hasDatabase) {
-      triggerDatabasePipeline(ctx, chartName, chartVersion)
+    String service = getServiceName(namespace)
+    if(service) {
+      if (hasDatabase) {
+        triggerDatabasePipeline(ctx, service, chartName, chartVersion)
+      }
+      triggerHelmPipeline(ctx, service, namespace, chartName, chartVersion)
     }
```
Comment (marctemp) [inline src/uk/gov/defra/ffc/ADO.groovy:11]: Let's log that the pipeline(s) aren't going to be ran for `namespace` (or sim.)

Comment (johnwatson484) [reply src/uk/gov/defra/ffc/ADO.groovy:13]: Agreed, done.

File: src/uk/gov/defra/ffc/ADO.groovy [hunk 2]
```diff
@@ -1,25 +1,41 @@
...
+      'ffc-demo': 'demo',
+      'ffc-pay': 'payments',
+      'ffc-ahwr': 'vetvisits',
+      'ffc-pr': 'pr',
+      'ea-wq': 'ea-wq',
+      'ffc-grants': 'grants',
+      'ffc-mpdp': 'mpdp'
+    ]
+
+    return services[namespace]
```
Comment (marctemp) [inline src/uk/gov/defra/ffc/ADO.groovy:27]: Correction: any language that uses backticks __OR__ the Elvis operator are ++ in my eyes

You took this use of Elvis away from me 😢 

BUT HERE IT IS ANYWAY

return services[namespace] ?: 'what happens when null'
return services[namespace]​ ?: {throw new RuntimeException('Unknown')}()

Kinda kewl, kinda just `??` in JS

**Review by marctemp — APPROVED** [review]
“Some people tap their feet, some people snap their fingers, and some people sway back and forth. I just sorta do ‘em all together, I guess.” - Elvis

Idk what the relevance is but approved

**Pull request merged**